### DISCUSSION TITLE
TURBO_INVOCATION_DIR in v2

### DISCUSSION THREAD

**Discussion by aldabil21** [vercel/turborepo#8578] [Help] [answered]
### Summary

We've been using `TURBO_INVOCATION_DIR` to run some build time scripts conditionally based on some core packages, it helps us to know which package is running the build script at the moment (It seems to be equal to `PROJECT_CWD` as well?)

We've upgraded to v2 today, and found out the `TURBO_INVOCATION_DIR` is no longer been set in the envs, which broke our build pipelines, it was confusing for a bit, then we figured it out. 

Perhaps it worth to be mentioned in the [upgrading guide](https://turbo.build/repo/docs/crafting-your-repository/upgrading#upgrading-to-20)? or better yet can be included in the codemod migration script?

**Comment by anthonyshew** [comment, marked as answer]
This variable is internal, which means it's not covered in semver or breaking changes. The documented System Environment Variables can be found here: https://turbo.build/repo/docs/reference/system-environment-variables
